contents/handbook/people/team-structure/platform.md

```diff
@@ -13,62 +13,48 @@ hideAnchor: true
 
 ## Mission
 
-Make using and developing for PostHog as reliable as running water. Wherever you want it.
-
-## Goals 
-
-- We don't lose events
-- Data is as up to date as possible
-- Engineers always be able to ship and build
-- Fail fast. Fix faster.
-- Ship anywhere
-- Stack scales with demand
-- Support Small Teams (and contributors) in building and debugging PostHog
-- Be frugal.
-
-## Responsibilities
-Concrete things we take responsibility over:
-
-- [app.posthog.com](https://app.posthog.com) and its infrastructure
-- On Prem & Single Tenant deployments
-- CI/CD - How we deploy
-- Data infrastructure (ClickHouse, Kafka)
-- Monitoring and Alerting stack
-- Event ingestion
-- Extensibility: Plugin server, plugins, and libraries
-
-## Customer
-
-- Other Small Teams in making sure they have the tools (databases, queues, etc) and the ability to deploy effortlessly that they need to build
-- End users (Both cloud and on-prem teams)
-
-## Output metrics
-
-### VPC
-###### Retention 
-- Metric: Retention
-- Objective: Better than cloud
-### Cloud
-###### Data Loss
-- Metric: Data loss %
-- Objective: < 0.1%
-###### Uptime 
-- Metric: Uptime
-- Objective: > 99.99%
-###### Speed 
-- Metric: Speed
-- Objectives
-  - Event ingestion: TBD
-  - Query response: TBD
-- Overall: We should anticipate increasing demand (either manually or automatically)
-##### Cost 
-- Metric: Infra Costs 
-- Objective: Our costs should grow at a rate that is sublinear relative to scale
-### Dev experience
-##### Dev experience NPS (Infra)
-- Metric: Developer experience (relating to infra) (maybe NPS?)
-- Objective: TBD (maybe NPS?)
+Make deploying, scaling and managing PostHog easy
+
+## Goals & Key Results
+
+* PostHog is the easiest self-hosted product to deploy and scale in the world
+    * Result: Volume of clients requring help @marcushyett-ph
+    * Result: % of successful installs / upgrades @guidoiaquinti
+    * Result: Max size of operating instances @fuziontech
+    * Montior: % of users on latest version of PostHog @tiina303
+    * Result: We don't lose customers because we can't scale enough (shared with Core Analytics) @marcushyett-ph
+* Move all posthog-supported instances to self-hosted
+    * Result: No alerts come to us from customer-instances
+* All valid events are ingested correctly
+    * Result: Time ingestion is down on cloud (from alert) @fuziontech
+    * Result: Lost events between ingress and clickhouse @fuziontech
+* Data is as up to date as possible
+    * Result: Latency between ingress and clickhouse @fuziontech
+* A good experience for plugin developers
+    * Result: Good docs, alerting, debugging and testing @yakkomajuri
```

Comment:
Isn’t the result just the number of plugins rises?

Reply:
Number of plugins is not something we have a lot of influence over as a team (e.g. we could just pay a ton of people to build plugins to meet the goal) - whereas getting the building experience better will make it easier / cheaper / faster to will lead to more higher-quality and maintainable plugins in the longer term.

Will also let @yakkomajuri chime in

Reply:
Yup, agree. Number of plugins is a larger goal that encompasses aspects of product, marketing, etc. The question is valid because one would think the same team owns both of these things, but as things currently stand, no team is _truly_ owning plugins, which is a topic that came up during our planning session and I believe we will be revisiting.

Reply:
we could still perhaps add some metrics to our graphs: number of plugins, % of users using plugins?

Reply:
I think we should track them somehow, but that's definitely different. I'm sold that the number of plugins is just the trailing metric from a good experience upstream.